libs/components/pages/src/lib/modules/page/page-links.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, inject } from '@angular/core';
+import { SkyLayoutHostService } from '@skyux/core';
+
+/**
+ * Displays page links within a block page layout.
+ * @internal
+ */
+@Component({
+  standalone: true,
+  selector: 'sky-page-links',
+  template: '<ng-content/>',
```

Comment:
Can we put a space before the `/`? I'm surprised Prettier didn't add it.

Reply:
Fixed.